### PR TITLE
Manager updates

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

File: src/assets/t9n/info-card/resources.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "fetchingData": "Fetching data",
   "edit": "Edit",
-  "enableEditing": "Editing must be enabled on the layer.",
+  "enableEditing": "Update cabability must be enabled on the layer.",
```
Comment (MikeTschudi) [inline src/assets/t9n/info-card/resources.json:4]: capability

File: src/assets/t9n/info-card/resources_en.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "fetchingData": "Fetching data",
   "edit": "Edit",
-  "enableEditing": "Editing must be enabled on the layer.",
+  "enableEditing": "Update cabability must be enabled on the layer.",
```
Comment (MikeTschudi) [inline src/assets/t9n/info-card/resources_en.json:4]: capability

**Review by MikeTschudi — APPROVED** [review]

File: src/assets/t9n/info-card/resources.json (hunk 1 above)
Comment (jmhauck) [reply src/assets/t9n/info-card/resources.json]: oops

**Comment by jmhauck** [comment]
#295

**Pull request merged**